packages/itwinui-css/src/bridge.scss

```diff
@@ -252,28 +284,101 @@
     // StrataKit input background colors are different from global values
     --_iui-field-color-background: var(--ids-color-bg-control-textbox);
     --_iui-field-color-background-hover: var(--_iui-field-color-background);
-    --_iui-field-color-background-disabled: var(--ids-color-bg-control-textbox);
+    --_iui-field-color-background-disabled: var(--_iui-field-color-background);
+  }
+
+  .iui-input-flex-container {
+    // StrataKit input background colors are different from global values
+    background-color: var(--ids-color-bg-control-textbox);
+  }
+
+  .iui-input-flex-container[data-iui-disabled='true'] {
+    // StrataKit disabled inputs don't have a different background color
+    --_iui-input-background-color: var(--iui-color-background);
+
+    // iTwinUI disabled icons don't change color, but StrataKit does. Thus, handling disabled case manually.
+    svg {
+      fill: var(--ids-color-icon-neutral-disabled);
+    }
+  }
+
+  /* ---------------------------------------------------------------------------- */
+  /* Label */
+
+  // StrataKit labels are slightly muted
+  .iui-input-label:not([data-iui-disabled='true']) {
+    --_iui-input-label-color: var(--ids-color-text-neutral-secondary);
+  }
+
+  /* ---------------------------------------------------------------------------- */
+  /* Popover */
+
+  .iui-popover-surface {
+    // StrataKit background color is different from global values
+    background-color: var(--ids-color-bg-elevation-level-1);
   }
 
   /* ---------------------------------------------------------------------------- */
-  /* ProgressRadial */
+  /* ProgressLinear, ProgressRadial */
 
   // StrataKit progress indicators's default color is not accent
   :is(.iui-progress-indicator-linear, .iui-progress-indicator-radial) {
     --_iui-progress-indicator-color-default-progress: var(--ids-color-icon-neutral-base);
   }
 
+  /* ---------------------------------------------------------------------------- */
+  /* SideNavigation */
+
+  // iTwinUI background color (--iui-color-background-backdrop)'s global mapping doesn't look correct in this context
+  .iui-side-navigation
+    .iui-sidenav-button:where([disabled], :disabled, [aria-disabled='true'], [data-iui-disabled='true']) {
+    background-color: var(--iui-color-background-disabled);
+  }
```

Comment:
Previously the SideNavigation looked more correct (see screenshot). We want the backgrounds of SideNavigation, Header and the main page to match.

![Screenshot](https://github.com/user-attachments/assets/592e1c9a-9b92-48e4-8cff-8a2318434513)